### PR TITLE
Added 2019 graduates to graduates.html

### PR DESCRIPTION
**Pull request by MustafaAlsaeedi**
#### Background Information

In response to "Add the Spring 2019 cohort section to the Graduates page #63"

#### Impact

Added new graduates to the graduates page.

#### Issue
Solves:  
#63

### CODE REVIEW

**Comment by brentonstrine** [comment]
Looks great, @MustafaAlsaeedi ! 

I have mentioned to others that I will intentionally not be doing code reviews. I want team members to do the code reviews! So you'll have to ask someone else to review. :D

File: graduates.html [hunk 1]
```diff
@@ -41,6 +41,237 @@ <h1 class="header-h1 grid-item">Refcode</h1>
       <h2>Refcode Graduates</h2>
       <p>Students who successfully completed all the course requirements are listed below. Professional referrals or references for any of the students listed below are available upon request.</p>
 
+      <h3>2019 Cohort</h3>
```
Comment (brentonstrine) [inline graduates.html]: Can you make this say "Spring 2019 Cohort" since we will have a Summer and a Fall cohort this year?

File: graduates.html [hunk 2]
```diff
@@ -41,6 +41,237 @@ <h1 class="header-h1 grid-item">Refcode</h1>
...
+
+      <figure class="graduates-picture2">
+        <img src="media/graduates&volunteers2019.jpg">
+        <figcaption>Graduates and volunteers from the 2019 coding class.</figcaption>
```
Comment (brentonstrine) [inline graduates.html]: Can you change this to say "Spring 2019 coding class" instead?

File: graduates.html [hunk 3]
```diff
@@ -41,6 +41,237 @@ <h1 class="header-h1 grid-item">Refcode</h1>
...
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Nisar</h4>
+        <div class="student-photo">
+          <img src="media/Nisar.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://nisar.ga/" rel="external">nisar.ga</a></li>
+          <li><a href="https://github.com/nisarfriend/NEW-Repository" rel="external">GitHub Profile</a></li>
```
Comment (brentonstrine) [inline graduates.html]: This link should go to the profile, not the repo. So just:

https://github.com/nisarfriend/

File: graduates.html [hunk 4]
```diff
@@ -41,6 +41,237 @@ <h1 class="header-h1 grid-item">Refcode</h1>
...
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Bizuayehu</h4>
+        <div class="student-photo">
+          <img src="media/Bizuayehu.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://bizuayehu.ga/" rel="external">bizuayehu.ga</a></li>
+          <li><a href="https://github.com/bizuayehu1968/Soli" rel="external">GitHub Profile</a></li>
```
Comment (brentonstrine) [inline graduates.html]: This should link to the profile not the repo, so: https://github.com/bizuayehu1968

File: graduates.html [hunk 5]
```diff
@@ -41,6 +41,237 @@ <h1 class="header-h1 grid-item">Refcode</h1>
...
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Selamawit</h4>
+        <div class="student-photo">
+          <img src="media/Selamawit.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://selamina.tk/" rel="external">selamina.tk</a></li>
+          <li><a href="https://github.com/selamcoding/selamina" rel="external">GitHub Profile</a></li>
```
Comment (brentonstrine) [inline graduates.html]: Just link to the profile.

**Comment by MustafaAlsaeedi** [comment]
I know, it can get overwhelming to review everyone's code. I have corrected the things you pointed and I will ask someone from the team to review it again.

**Comment by brentonstrine** [comment]
Ah, this code still isn't merged! @MustafaAlsaeedi you will have to be more proactive in getting a reviewer.

**Comment by MustafaAlsaeedi** [comment]
You are right, @brentonstrine. I will ask one of the developers to review it since the advisors are not doing it going forward.

**Pull request merged**